pyproject.toml

```diff
@@ -10,7 +10,8 @@ readme = "README.md"
 requires-python = ">= 3.8.1"
 dependencies = [
     "guardrails-ai>=0.4.0",
-    "llama-index>=0.10.3"
+    "llama-index>=0.10.3",
+    "nltk>3.0, <=3.8.1"
```

Comment:
this needs to happen for now or itll cause a incompatibility with oss and errors with other validators if theyre installed in the wrong order expecting the older nltk and punkt